config.json.sample

```diff
@@ -3,9 +3,9 @@
     "api_url": "https://portal.trinitycyber.com/graphql",
     "api_key": "<add me>",
     "marker_file": "~/.tc3/portal-s3-after-marker.txt",
-    "customer_gids": [],
```

Comment:
Can we tell from FCP logs if customers are using this field today?

Reply:
Just one.   This is also just deprecating it and removing it from the sample config.   It will continue to work.   I just want new users to switch to using the new filter.